version.go

```diff
@@ -1,4 +1,15 @@
 package specs
 
+import "fmt"
+
+const (
+	// VersionMajor is for an API incompatible changes
+	VersionMajor = 0
+	// VersionMinor is for functionality in a backwards-compatible manner
```

Comment:
Maybe “VersionMinor is for backwards-compatible API changes”?  And then mirror that for `VersionMajor` with “VersionMajor is for backwards-incompatible API changes”.